README.md

```diff
@@ -61,6 +61,7 @@ Gazebo Classic and Ignition software.
         1. Classic release: `<path to release-tools>/release.py -r 1 gazebo11 11.2.0 secrettoken --dry-run`
     1. If the dry run succeeds, run the same command again, now without `--dry-run`.
 1. Check that:
-    * Several `-debbuilder` jobs have been queued in https://build.osrfoundation.org/
+    * Several `-debbuilder` jobs have been queued in https://build.osrfoundation.org/ and watch those jobs to see if any of them fail.
     * A pull request was opened to https://github.com/osrf/homebrew-simulation
         1. This pull request may take a minute or two to open.
+        1. Once it is open, make a comment containing the text "build bottle"
```

Comment:
Should we add an example link? Something like this: https://github.com/osrf/homebrew-simulation/pull/1225#issuecomment-742870169

Also, should we mention what you should check for/expect after commenting "build bottle"?

Reply:
I've added a link to the osrf/homebrew-simulation README in 9cc73a90070cebaec8ec82c3d4db825ce6a11761, since I think it will be easier to keep that info over there.

> should we mention what you should check for/expect after commenting "build bottle"?

can you look at the linked readme and see if that answers your question? if not, let's follow up in that other repo